src/views/VideoView/VideoView.tsx

```diff
@@ -70,6 +78,24 @@ const VideoView: React.FC<RouteComponentProps> = () => {
     })
   }, [addVideoView, videoID])
 
+  // Save the video timestamp
+  useInterval(() => {
```

Comment:
I think there may be a better approach to this. Now we're passing the video component ref and accessing it's internal state. In general that should be avoided if possible. Instead, we could have some callback (e.g. `onTimeUpdated`) on the video component and just pass some function to it that would update the saved state. It can be debounced so it doesn't save too often.